common/static/js/capa/jsinput/jsinput_example.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Dropdown with Dynamic Text</title>
+    <link rel="stylesheet" type="text/css" href="jsinput_example.css">
+  </head>
+  <body>
+    <script src="https://files.edx.org/custom-js-example/jschannel.js"></script>
+    <script src="jsinput_example.js" defer></script>
+    <label class="directions">Choose the correct choice:
```

Comment:
@cahrens Agh, sorry I missed this earlier. "Choose the correct choice:" ->
"Select an option from the list:"
(Unless you specifically wanted people to choose "correct"??)